### PR TITLE
Fixes #263

### LINKED ISSUE #263: When click is held down over a drop down, the text becomes highlighted
**Priority:** P5
**Problem:** The highlighted text isn't aesthetically pleasing
**Pictures:**
pivotal_network_normal
pivotal_network_hold_down_click_on_dropdown

### PR DESCRIPTION
**Pull request by RochesterinNYC**
The dropdown component automatically seems to use button groups, which will turn blue when any button in the group is clicked and held on (or for a brief moment when it is clicked and considered active). This commit removes the background so no blue flashing/background color appears.

### CODE REVIEW

**Comment by atomanyih** [comment]
@RochesterinNYC This would be a design change to both dropdowns and button groups, so we probably want @pivotal-plech to weigh in.

The button group active state now looks like this:
screen shot 2015-10-08 at 2 22 12 pm

The same applies to dropdowns:
screen shot 2015-10-08 at 2 26 34 pm

The contrast is all wonked, so we'd probs have to change the link color too.

A better solution might be to make it easier for users to specify colors. Thoughts?

**Comment by pivotal-plech** [comment]
TL;DR
This is the desired design:
image

---

To attempt to clarify. The following was the desired `.active` state per the design:
image

However, the `.active` state was supposed to be a persistent active class. The :active pseudo class for the button groups should be consistent with the :active pseudo class for the default button. The previous attachment was a miscommunication of `.active` state...

@RochesterinNYC ping me if you have any questions. That's a really confusing paragraph.

**Comment by atomanyih** [comment]
@pivotal-plech any thoughts as to how this would affect #263

**Comment by pivotal-plech** [comment]
@atomanyih In an ideal alternate reality, the link dropdown wouldn't inherit the :active state from buttons but would instead have the same :active state as specified here:

image

**Comment by atomanyih** [comment]
@pivotal-plech Yeah that makes a lot more sense. IDG why bootstrap makes
dropdowns button groups but hey

If you put a story in the backlog I'll pick it up when I'm next working on
PUI

**Comment by atomanyih** [comment]
I just fixed #263, so this probs isn't relevant anymore
